test/functional/test_framework/script_util.py

```diff
@@ -25,6 +35,10 @@
 DUMMY_P2WPKH_SCRIPT = CScript([b'a' * 21])
 DUMMY_2_P2WPKH_SCRIPT = CScript([b'b' * 21])
 
+def key_to_p2pk_script(key, main = False):
```

Comment:
what is `main`?

Reply:
It seems that the `main` parameter for the script_util helpers is only used in `test_framework/address.py` where the mainnet boolean parameters is passed-through (for whatever reason...). Since P2PK addresses don't exist, it can be removed. Done.